### DISCUSSION TITLE
How can I set ARM to ignore data Blurays?

### DISCUSSION THREAD

**Discussion by phirestalker** [automatic-ripping-machine/automatic-ripping-machine#578] [Q&A] [answered]
I am ripping my PS3 game discs with a program called libray. This is the same system I use for ripping movies, so ARM is set to launch when the disc is inserted. I would like to disable this until I have time to script something up.

I looked through the config file and found no settings for disabling data disc ripping. Is there a way?

**Comment by 1337-server** [comment, marked as answer]
It's not currently possible
